src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -623,7 +738,7 @@ public static bool HaveSameReturnTypes(Symbol member1, TypeMap? typeMap1, Symbol
                     true);
         }
 
-        private static bool HaveSameConstraints(Symbol member1, TypeMap? typeMap1, Symbol member2, TypeMap? typeMap2)
+        private static bool HaveSameConstraints(Symbol member1, TypeMap? typeMap1, Symbol member2, TypeMap? typeMap2, bool ignoreNullability = true)
```

Comment:
>bool ignoreNullability = true [](http://example.com/codeflow?start=118&length=29)

I think it would be better to make this parameter required here and in other signature where we add it.